VivianDoreen/store-manager-frontend
# #164349666 Setup an empty react project

## What does this PR do?
This PR sets up an empty react project.

## Description of Task to be completed?
Setup webpack
Setup Babel

## How should this be manually tested?
run the following commands

- git clone https://github.com/andela/Ah-frontend-guardians.git

- git checkout ch-empty-react-project-162685616
- npm install (to install all the dependencies in the package.json)
- npm start
- Visit https://localhost:8080/

you should see this image

![image](https://user-images.githubusercontent.com/41106626/53681407-5eb63200-3cfa-11e9-8109-e48233991781.png)

What are the relevant pivotal tracker stories?
[#164349666](https://www.pivotaltracker.com/story/show/164349666)
